<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/new/prefix1/console.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>50</y>
      <width>125</width>
      <height>245</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(158, 172, 136);
border: 2px solid black;
border-radius: 0;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 7;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 7;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="resetLabel">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>330</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: none;
color: rgba(33, 33, 33, 233);
font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: none;
color: rgba(33, 33, 33, 233);
font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreData">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>256</y>
      <width>33</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="levelLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLabel" name="levelData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>226</y>
      <width>33</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="recordLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>266</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>record:</string>
    </property>
   </widget>
   <widget class="QLabel" name="recordData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136);
font: 10pt &quot;.AppleSystemUIFont&quot;;
image: none;
color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actionBtn">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>405</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 35;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upBtn">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>374</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 25;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rightBtn">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 25;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dropBtn">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>445</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 25;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leftBtn">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 25;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="nextView">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>67</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136)</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="holdView">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>165</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 172, 136)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(251, 214, 69);
	image: none;
	border: 3px solid rgba(94, 94, 94, 208);
	border-radius: 7;
}
QPushButton:pressed {
	background-color:  rgb(255, 147, 0);
	border: 3px solid rgb(0, 0, 0);
	image: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pauseLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: none;
color: rgba(33, 33, 33, 233);
font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameOverLabel">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: none;
font: 700 18pt &quot;Marker Felt&quot;;
color: rgb(40, 45, 20);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
